packages/disploy/src/structs/ChatInputInteractionResolvedOptions.ts

```diff
@@ -0,0 +1,58 @@
+import { APIGuildChannel, ChannelType, Snowflake } from "discord-api-types/v10";
+import type { BaseChannel } from "./BaseChannel";
+import type { ChatInputInteraction } from "./ChatInputInteraction";
+import { GuildTextChannel } from "./GuildTextChannel";
+import { GuildVoiceChannel } from "./GuildVoiceChannel";
+import { PartialGuildMember } from "./PartialGuildMember";
+import { User } from "./User";
+
+export class ChatInputInteractionResolvedOptions {
+    public interaction: ChatInputInteraction;
+    public members: Map<Snowflake, PartialGuildMember>;
+    public users: Map<Snowflake, User>;
+    public channels: Map<Snowflake, BaseChannel>
+    constructor(interaction: ChatInputInteraction) {
+        /**
+         * The interaction the resolved options belong to.
+         */
+        this.interaction = interaction;
+        
+        /**
+         * The resolved members.
+         */
+        this.members = new Map()
+        /**
+         * The resolved users.
+         */
+        this.users = new Map()
+        /**
+         * The resolved channels.
+         */
+        this.channels = new Map()
```

Comment:
This is not how JSDoc comments work, you need to put them above the properties definition, not the declaration.